### ISSUE TITLE
Wordpress dashboard stuck on loading using mod_http2

### ISSUE THREAD

**Issue by frk7z** [oerdnj/deb.sury.org#649]
When I try to login into WP admin dashboard using apache2 with mod_http2 from your repository on Ubuntu 16.04, the browser keeps loading forever.  Apache2 uses mpm_event with php-fpm and ssl self-signed. Mods enabled are mod_proxy, mod_proxy_fcgi, mod_ssl and mod_rewrite.

My config is here, please tell me if anything is wrong.

```
<VirtualHost *:80>
    ServerName wptest.test
    Redirect / https://wptest.test/
</VirtualHost>

<VirtualHost *:443>
    ServerName wptest.test
    Protocols h2 http/1.1
    DocumentRoot /var/www/wptest.test/htdocs
    DirectoryIndex index.php index.html

    SSLEngine on
    SSLCertificateFile "/etc/apache2/selfcerts/wptest.test.crt"
    SSLCertificateKeyFile "/etc/apache2/selfcerts/wptest.test.key"

    SSLCipherSuite EECDH+ECDSA+AESGCM:EECDH+aRSA+AESGCM:EECDH+ECDSA+SHA256:EECDH+ECDSA+SHA384:EECDH+aRSA+SHA256:EECDH+aRSA+SHA384:EECDH+AES128:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!SRP:!DSS:!RC4:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA:!CAMELLIA

    <Directory "/var/www/wptest.test/htdocs">
        AllowOverride All
    </Directory>

    <FilesMatch "\.php$">
        SetHandler "proxy:fcgi://127.0.0.1:9000"
    </FilesMatch>

    LogLevel http2:trace2
    ErrorLog ${APACHE_LOG_DIR}/wptest.test-error.log
    CustomLog ${APACHE_LOG_DIR}/wptest.test-access.log combined
</VirtualHost>
```

Here the error-log file using trace2 if helps in any way.

[wptest.test-error.log.txt](https://github.com/oerdnj/deb.sury.org/files/1152063/wptest.test-error.log.txt)

PD: sorry for my english

**Comment by ConteZero** [comment]
Perhaps it is related to
https://github.com/icing/mod_h2/issues/143

**Comment by frk7z** [comment]
I do not know if they were related, but since that "ticket" was solved using the new version 1.10.10 of mod_http2 😄

**Comment by ConteZero** [comment]
I think that after latest apache package update this issue could be closed.
